smartcards/card.py

```diff
@@ -83,6 +83,9 @@ def write_short_and_long_values(self, short_value_bytes, long_value_bytes):
             chunk_num += 1
             offset_into_bytes += self.CHUNK_SIZE
 
+        # wait a little bit
```

Comment:
nit: would be good to note what process is being waited on.

Reply:
for speed, I'm going to merge this, but quick note on this: a read soon after a write doesn't read the right data. It's either a firmware or EEPROM thing with the smartcards. The data needs to "settle" I guess.